internal/web/client_install_handler.go

```diff
@@ -8,7 +8,7 @@ import (
 
 // DefaultSidecarProxyAddr is the default location where a client can
 // download the sidecar proxy container image.
-var DefaultSidecarProxyAddr = "127.0.0.1:5000/sidecar-proxy:latest"
+var DefaultSidecarProxyAddr = "127.0.0.1:5000/sidecar-proxy:1.0.0"
```

Comment:
Again, can we pull version out of the code?

Reply:
This value is always overwritten. We could leave it as "latest"